code/game/gamemodes/meteor/meteor.dm

```diff
@@ -69,6 +69,7 @@
 		universe.OnExit()
 
 	var/datum/universal_state/meteor_storm/meteor_storm = new /datum/universal_state/meteor_storm
+	universe = meteor_storm
```

Comment:
I wanted to pass vars along with it and nothing fucking worked so I simply did a thing and it worked fine